test/models/post_test.rb

```diff
@@ -1,17 +1,31 @@
 require 'test_helper'
 
 class PostTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
 
   test "Add Post" do
+    post = Post.new({
+        title: "Hello World",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.